src/classes/Gifts.cls

```diff
@@ -54,8 +54,24 @@ public with sharing class Gifts {
         return giftsToReturn;
     }
 
+    public void giftsToProcessingStatus() {
+        giftsToUpdate.clear();
+        for (DataImport__c gift : dataImports) {
+            if (isProcessableGift(gift)) {
+                gift.Status__c = BDI_DataImport_API.bdiProcessing;
```

Comment:
@pedrodalcol @lparrott @callen-sfdo - just want to give the heads up that we are introducing the "Processing" status for DI here! Should not impact anyone as this PR are only sets the value within the context of Gift Entry